docs/Limitations.md

```diff
@@ -216,6 +218,16 @@ Equivalent functionality can be achieved with the newer docker networking comman
 See more documentation at
 [docs.docker.com](https://docs.docker.com/engine/userguide/networking/default_network/dockerlinks/).
 
+## Storage limitations
+
+### Subpaths
```

Comment:
I suggest to expand the title to something like "Kubernetes volumeMounts subPath".  More or less how it is called in https://kubernetes.io/docs/concepts/storage/volumes/#using-subpath

Reply:
Good idea. Done.